devs/docs/es-backports.rst

```diff
@@ -234,9 +234,9 @@ should be crossed out as well.
 - [ ] 27067de6991 Make MappedFieldType#meta final (#59383)
 - [x] b599f7a9c07 Fix estimate size of translog operations (#59206)
 - [ ] 2b70758a052 Correct type parametrization in geo mappers. (#59583)
-- [ ] cc7093645cc Cleanup some Serialization Code around Snapshots (#59532) (#59606)
-- [ ] b8d73a1e7ee Default gateway.auto_import_dangling_indices to false (#59302)
-- [ ] 691759fb1fc Validate snapshot UUID during restore (#59601)
+- [x] cc7093645cc Cleanup some Serialization Code around Snapshots (#59532) (#59606)
```

Comment:
Skipped the whole package `action/admin/cluster/snapshots/status` from the referenced commit as we don't have such code in our repo